libminifi/test/unit/FileUtilsTests.cpp

```diff
@@ -477,3 +477,27 @@ TEST_CASE("FileUtils::get_relative_path", "[TestGetRelativePath]") {
   REQUIRE(*FileUtils::get_relative_path(path, base_path / "") == std::filesystem::path("subdir") / "file.log");
   REQUIRE(*FileUtils::get_relative_path(base_path, base_path) == ".");
 }
+
+TEST_CASE("file_clock to system_clock conversion tests") {
+  using namespace std::chrono;
+
+  static_assert(system_clock::duration::period::num == file_clock::duration::period::num);
+  constexpr auto lowest_den = system_clock::period::den < file_clock::period::den ? system_clock::period::den : file_clock::period::den;
```

Comment:
`std::min` doesn't work here?

Reply:
sure thing, good idea, I've changed it in https://github.com/apache/nifi-minifi-cpp/pull/1560/commits/41f3dd4ef1416a1bf5ab38d7052e08678adc2d88